frontend/ui/data/css_tweaks.lua

```diff
@@ -613,6 +613,22 @@ This is just an example, that will need to be adapted into a user style tweak.]]
         },
         {
             title = _("In-page footnotes"),
+            {
+                id = "footnote-inpage_fb2";
+                title = _("In-page FB2 footnotes"),
```

Comment:
Btw, did you see my comment on MR about being in favor of enabling these by default?

Reply:
Yes I did. And you already mentionned that nearly a year ago.
See (again) my answer at https://github.com/koreader/koreader/issues/5015#issuecomment-491292942.
(Nothing much to add or remove from it one year later :| )